### ISSUE TITLE
items listed under the "Charts & Stats" and then "Items required to level all owned crew" should be sorted alphabetically

### ISSUE THREAD

**Issue by badrobit** [stt-datacore/website#268]
when going through the list of items and checking for which ones we have it is much easier to process if they are stored in the same way as the game (alphabetically) that way we aren't always jumping all over the place within the list to find out if we can convert specific items into replicator rations or not.

**Comment by ineffyble** [comment]
Ooh, so you're wanting to identify components where you have more than you currently need?

**Comment by ineffyble** [comment]
If so, I think a better approach would be to add that as a new category under Unneeded Items

**Comment by badrobit** [comment]
I agree having it as another entry under unneeded items would be great, but it would also need to show the number its okay to remove as I think stuff from that list already shows up once it is entirely unneeded.
